### PR TITLE
Fix: Hovering Card's Edge causes animation loop

### PR DESCRIPTION
**Pull request by sshanzel**
Hi Team,

I've created a `wrapper` for the `card` so that the browser won't be confused about which element has currently hovered, then the animation will be performed on the wrapped `card` so the hover trigger point will still be on the same position.

If you have any concerns, please let me know so I can make the necessary changes.

Thanks!

Best Regards,
Hansel

### CODE REVIEW

**Comment by sshanzel** [comment]
Looks like some existing tests failed. Imma check it out.

**Comment by sshanzel** [comment]
I'm not sure if this error in the logs has something to do it.

image

**Comment by sshanzel** [comment]
My bad, gotta check the errors.

**Comment by idoshamun** [comment]
Hi man!

Great to see you contributing again 🎉 
Usually, I'm not a big fan of wrapping an existing element with a div as it may break some selectors and the app behavior.
In this case I understand that it might require to do, but I would like to kindly ask you if you can give it one more chance to find another way that doesn't involve wrapping the element. What do you think?

**Comment by sshanzel** [comment]
Actually that's also my concern since the beginning, but the way i see it, there must be an element which won't be moving along with the card itself. Either wrap the card as a whole or wrap the elements inside of it (then the selectors may get into trouble) which i felt might bring more unexpected behaviours. Though, I'd still look out for alternatives and let you know what i can find. Thanks a lot for the review!! :grin:

**Comment by sshanzel** [comment]
Hi @idoshamun, I tried using a `pseudo-element` to try and implement the `hover` feature in it but as a `pseudo-element`, its current `position` is `relative` to its actual `element`. I can't also put a wrapper `element` from the inside as it would also move with its parent. 

I've also thought of the `pseudo-element` to have a 100% `height` + the value for `transform: translatey` (`4px`) but it would still yield the loop after hovering down `4px` more.

Any thoughts on this? I'm really not a fan of a wrapper div as it may result to breaking change on pre-existing selectors, but i think this really has to come from the outside.

**Comment by idoshamun** [comment]
The `pseudo-element` sounds like a good direction, I thought about it as
well.
What if we apply an inverse transform to this `pseudo-element` so even if
the card is hovered it will remain still?

**Comment by sshanzel** [comment]
I've actually thought of this but didn't actually believe in it, tried it now, and worked! I thought that if you hover down a bit more (the amount of inversed transform) it would still yield the loop as `pseudo-element` is relative to its original `element`. Such insolence of mine! Lol.

**Comment by sshanzel** [comment]
This fact actually annoys meself. Another committed poor decision-making. Lol. Ughhh

**Comment by idoshamun** [comment]
Haha don't be too tough with yourself. Is it ready for review?

**Comment by sshanzel** [comment]
Hahaha. Yes bro I think this is now ready for review. Thanks a lot!!!

File: packages/components/src/components/DaCard.vue [hunk 1]
```diff
@@ -127,6 +127,13 @@ export default {
   & .ls-blur-up-img.ls-inview.ls-original-loaded {
     opacity: 0;
   }
+
+  &:before {
+    position: absolute;
+    content: '';
+    height: 100%;
+    width: 100%;
+  }
```
Comment (idoshamun) [inline packages/components/src/components/DaCard.vue:136]: I think it fits better inside the `.animate-cards` selector.
If the cards animation is off there is no transition and this pseudo-element is not needed.
Anyway, this works great! 🥳

Comment (sshanzel) [reply packages/components/src/components/DaCard.vue:136]: Definitely true on that.

**Comment by sshanzel** [comment]
Hi @idoshamun, i've made the necessary changes. kindly check if it looks alright now. Thanks a lot for the review! 🥳

**Pull request merged**